build_tools/packaging/python/templates/rocm/src/rocm_sdk/tests/core_test.py

```diff
@@ -33,7 +33,7 @@
     ("amdclang++", ["--help"], "clang LLVM compiler", True),
     ("amdclang-cpp", ["--help"], "clang LLVM compiler", True),
     ("amdclang-cl", ["-help"], "clang LLVM compiler", True),
-    ("amdflang", ["--help"], "clang LLVM compiler", True),
+    ("amdflang", ["--help"], "flang LLVM compiler", True),
```

Comment:
How about we just check for "LLVM compiler" so both work? Not sure if Windows and Linux not matching is expected, this could be hiding a bug.
```suggestion
    ("amdflang", ["--help"], "LLVM compiler", True),
```